bootstrap.py

```diff
@@ -22,7 +22,6 @@
 import shutil
 import sys
 import tempfile
-
```

Comment:
bootstrap.py is obsolete and doesn't work with modern setuptools versions.  People are advised to pip install zc.buildout somewhere and then run `buildout bootstrap` instead.

We've removed bootstrap.py from many other repositories and I recommend removing it here too.

Reply:
it will be interesting to learn this. will put it on my docket for some time soon.

Reply:
I think we should just drop buildout and only have a tox setup. People can then use `venv` and `buildout if they want to.